README.md

```diff
@@ -44,35 +44,59 @@ services:
 services:
   db-mariadb:
     image: mariadb
-    restart: always
+    restart: unless-stopped
+    ports:
+      - "3306:3306"
     environment:
       MYSQL_ROOT_PASSWORD: example
       MYSQL_DATABASE: example
       MYSQL_USER: example
       MYSQL_PASSWORD: example
-      MYSQL_ALLOW_EMPTY_PASSWORD: yes # no
-      MYSQL_RANDOM_ROOT_PASSWORD: yes # <not set>
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'no'
+      MYSQL_RANDOM_ROOT_PASSWORD: 'no'
+    volumes:
+      - mariadb:/var/lib/mysql
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    restart: unless-stopped
+    ports:
+      - "8080:80"
```

Comment:
huh, looks problematic, but I hope everyone will see port 80 and will change it

Reply:
80 is inside the container. Do you mean 8080 being too used for other apps?

Reply:
Ah, yeah, 8080, but never mind.